lib/plottr_components/src/components/tag/TagListView.js

```diff
@@ -3,44 +3,141 @@ import PropTypes from 'react-proptypes'
 import { Glyphicon, Nav, NavItem, Button } from 'react-bootstrap'
 import cx from 'classnames'
 import { t as i18n } from 'plottr_locales'
-import { initialState } from 'pltr/v2'
 import UnconnectedSubNav from '../containers/SubNav'
 import UnconnectedTagView from './TagView'
-
-const { tag } = initialState
+import UnconnectedTagCategoriesModal from './TagCategoriesModal'
 
 const TagListViewConnector = (connector) => {
   const SubNav = UnconnectedSubNav(connector)
   const TagView = UnconnectedTagView(connector)
+  const TagCategoriesModal = UnconnectedTagCategoriesModal(connector)
 
   class TagListView extends Component {
     state = {
       appending: false,
+      categoriesDialogOpen: false,
+      tagDetailId: null,
+      newCategoryId: null,
+    }
+
+    static getDerivedStateFromProps(props, state) {
```

Comment:
This and the next static function should be selectors.  (That way other apps, like mobile, can use the same wiring to get tag categories etc.)